model/event/task_event.go

```diff
@@ -114,13 +114,11 @@ func LogTaskStarted(taskId string, execution int) {
 	logTaskEvent(taskId, TaskStarted, TaskEventData{Execution: execution})
 }
 
-func LogTaskFinished(taskId string, execution int, hostId, status string) {
-	logTaskEvent(taskId, TaskFinished, TaskEventData{Execution: execution, Status: status})
-	LogHostEvent(hostId, EventTaskFinished, HostEventData{TaskExecution: execution, TaskStatus: status, TaskId: taskId})
-}
-
-func LogDisplayTaskFinished(taskId string, execution int, status string) {
```

Comment:
I don't understand the rationale for removing this. When all execution tasks in a display task finish, it makes sense to be logging a task event but not a host event. What was the flaw in logic from before?

Reply:
I think having a separate function for this is unnecessary because we call LogTaskFinished in many places, but if its a display task we don't want to log a host event ever (because display tasks don't even have hosts). 

So I moved this functionality into LogTaskFinished (by checking if isDisplay is true, although I guess I could check hostId == "" instead actually) to cover all cases and not just that one place where LogDisplayTaskFinished was originally called.

Reply:
tl;dr the functionality is the same in the place that function was being called